tools/quota-manager/Pipfile.lock

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
Pretty much looks good but this file is not needed, it's generated by pipenv for your installation, can you add a .gitignore file with this file in it and amend the changes please ? Ones done i will approve this.

Reply:
Lock ensures the integrity. If someone hits a compromised pip server, sha digests in the lock are used to detect it.  See [the advise](https://pipenv-fork.readthedocs.io/en/latest/basics.html#general-recommendations-version-control)

can I keep it?

Reply:
SGTM